src/schemas.py

```
@@ -720,6 +750,16 @@ class TriggerDreamRequest(BaseModel):
         None, description="Observed peer name (defaults to observer if not specified)"
     )
     dream_type: DreamType = Field(..., description="Type of dream to trigger")
+    session_id: str = Field(..., description="Session ID to scope the dream to")
+    reasoning_focus: Literal["deduction", "induction", "knowledge_update"] | None = (
+        Field(
+            None,
+            description="Optional focus mode to bias the dream toward specific reasoning: "
+            + "'deduction' prioritizes logical inferences from explicit facts, "
```

Ruff (0.14.10)

Explicitly concatenated string should be implicitly concatenated

Remove redundant '+' operator to implicitly concatenate

(ISC003)